doc/user/content/cli/_index.md

```diff
@@ -30,6 +30,19 @@ Flag | Default | Modifies
 `-v` | N/A | Print version and exit
 `-vv` | N/A | Print version and additional build information, and exit
 
+If a command line flag named above takes an argument, you can alternatively set
+that flag via an environment variable named after the flag. For example, to set
+the data directory, you can set the `MZ_DATA_DIRECTORY` environment variable. If
```

Comment:
I guess I'd put this in its own `p` after everything else but before `Note that command line flags...`? After doing the other shuffles, trust your heart about where this ought to go.

Reply:
Done!